tsl/src/chunk_copy.c

```diff
@@ -624,13 +817,185 @@ chunk_copy(Oid chunk_relid, const char *src_node, const char *dst_node, bool del
 		ErrorData *edata;
 		MemoryContextSwitchTo(oldcontext);
 		edata = CopyErrorData();
-		edata->hint =
-			psprintf("chunk copy operation id: %d (%s).", cc.fd.id, NameStr(cc.fd.operation_id));
+		edata->hint = psprintf("chunk copy operation id: (%s).", NameStr(cc.fd.operation_id));
```

Comment:
nit: I guess we can remove `()` as well or use quotes

Reply:
done